tensorflow/stream_executor/cuda/cuda_gpu_executor.cc

```diff
@@ -1129,6 +1129,30 @@ DeviceDescription *CUDAExecutor::PopulateDeviceDescription() const {
           CU_DEVICE_ATTRIBUTE_MAX_REGISTERS_PER_MULTIPROCESSOR, device_)
           .ValueOrDie());
 
+  const char* blank_ptx =
+      ".version 6.0\n"
+      ".target sm_30\n"
+      ".address_size 64\n"
+      "\n"
+      "        // .globl       _Z6ValAddPf\n"
+      ".visible .entry _Z6ValAddPf(\n"
+      ")\n"
+      "{\n"
+      "        ret;\n"
+      "}\n";
+  const char* kernel_name = "_Z6ValAddPf";
```

Comment:
Do we need to use a C++ mangled name?  Like, could we just use "test" or something?